index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const app = express()
+const axios = require('axios')
+app.use(express.json())
+
+app.get('/getWeather', function(req, res){
+    const url = 'https://codubee-projects-api.herokuapp.com/getWeather'
+    axios.get(url)
+        .then((response) => {
+            console.log(response.data)
+            res.status(200).json({"Temperature is: ": response.data + " F"})
```

Comment:
Can the message here just be "temp":"76". The key "Temperature is:" is a bit of an odd key to have.